error_handler.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+import io
+import logging
+import traceback
+import urllib
+
+import discord
+from discord.ext import commands
+
+logger = logging.getLogger(__name__)
+
+
+def get_full_class_name(obj):
```

Comment:
This function is very confusing to look at, i think it needs a doc comment for clarity